### ISSUE TITLE
Progressive Load Fails to Work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#205]
```
What steps will reproduce the problem?
1. Modified all code to hard code progressiveLoad as true.
2. Verify pdf2swf.exe used the -T 9 switch
3. Open the 500 page PDF and forced to wait for a full load.

What is the expected output? What do you see instead?
I expected the progressive load to take place and to begin seeing pages about 
the time it hit 15-25% loaded.

What version of the product are you using? On what operating system?
FlexPaper_1.4.2_SDK_3.5_swc.zip on Windows

Please provide any additional information below.

```

Original issue reported on code.google.com by `r...@hines57.com` on 21 Jun 2011 at 2:22

**Comment by GoogleCodeExporter** [comment]
```
please provide a sample PDF/SWF which can be used to reproduce this? As well as 
information about which version of SWFTools that was used to produce it
```

Original comment by `erik.eng...@devaldi.com` on 21 Jun 2011 at 9:00

**Comment by GoogleCodeExporter** [comment]
```
also try the latest version (1.4.5) and see if the problem still persists
```

Original comment by `erik.eng...@devaldi.com` on 21 Jun 2011 at 9:01

**Comment by GoogleCodeExporter** [comment]
```
swftools version 0.9.1

Conversion Command-line: pdf2swf.exe -s poly2bitmap -f -T 9 -t -s 
storeallcharacters "D:/share/jobs/millington_high_school/spec/revision_1/642 
Page Test.pdf" -o 
"c:/web/planroom/public/swf/bWlsbGluZ3Rvbl9oaWdoX3NjaG9vbC9zcGVjL3JldmlzaW9uXzEv
NjQyIFBhZ2UgVGVzdC5wZGY=.swf"

Attempted again this morning with 1.4.5 (Commercial License) source compiled in 
Flex 4.5 and no go. FlexPaper instantiation looks like this:

<?xml version="1.0" encoding="utf-8"?>
<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml"
                xmlns:fp="com.devaldi.controls.flexpaper.*"
                width="100%" height="100%" backgroundAlpha="0.0" backgroundColor="#F70E0E"
                horizontalAlign="left" initialize="initVars()" layout="absolute" verticalAlign="top"
                >

    <mx:Script>
        <![CDATA[
            [Bindable]
            public var authToken:String;

            private function initVars():void {              
                pdfViewer.SwfFile = '/swf/' + Application.application.parameters.authToken + '.swf';
                pdfViewer.ProgressiveLoading = true;
                //pdfViewer.ReadOnly = true;
                //pdfViewer.SearchToolsVisible = false;
                //pdfViewer.Scale = 0.6;
                //pdfViewer.ZoomTransition = "easeOut";
                //pdfViewer.ZoomTime = 0.5;
                //pdfViewer.ZoomInterval = 0.1;
                //pdfViewer.FitPageOnLoad = false;
                //pdfViewer.FitWidthOnLoad = true;
                //pdfViewer.PrintDialogEnabled = false;
                //var menu:ContextMenu = new ContextMenu();
                //menu.hideBuiltInItems();
                //this.contextMenu = menu;
            }
        ]]>
    </mx:Script>

    <fp:FlexPaperViewer id="pdfViewer" x="0" y="0" width="100%" height="100%">

    </fp:FlexPaperViewer>
</mx:Application>

I'm trying to attach the PDF but seems like it may be to big. Please forgive a 
second post if necessary.
```

Original comment by `r...@stechstudio.com` on 22 Jun 2011 at 3:26

**Comment by GoogleCodeExporter** [comment]
```
This is the PDF we test against. This PDF would load progressively when we used 
1.4.1.

http://support.reproconnect.com/uploads/642%20Page%20Test.pdf

Earlier, I meant to state Flex Builder 4.5 w/Flex SDK 3.5. Here is a screenshot 
of my Flex Compiler Settings: 
http://support.reproconnect.com/uploads/FlexPaper1.4.5-CompilerSettings.png


```

Original comment by `r...@stechstudio.com` on 22 Jun 2011 at 3:46

**Comment by GoogleCodeExporter** [comment]
```
Downloaded  1.4.1 (Commercial License) source and compiled it with Flex Builder 
4.5 w/Flex SDK 3.5 - the exact same configuration as 1.4.5 - and progressive 
loading works just fine with this test PDF.
```

Original comment by `r...@hines57.com` on 23 Jun 2011 at 2:29

**Comment by GoogleCodeExporter** [comment]
```
I spoke too soon, the progressive loading with 1.4.1 appears to work just fine, 
but after further testing, it seems to "stick" at 100% loaded tho appears to 
continue to be loading. Scrolling through the document demonstrates a lot of 
"blank" pages that shouldn't be, and eventually lost all text on all pages.
```

Original comment by `r...@hines57.com` on 23 Jun 2011 at 2:41

**Comment by GoogleCodeExporter** [comment]
```
We reverted all our code back to 1.4.1 and have things working as advertised 
again. Once Progressive Loading gets sorted in 1.4.5+ we would love to get 
upgraded.
```

Original comment by `r...@hines57.com` on 24 Jun 2011 at 5:21

**Comment by GoogleCodeExporter** [comment]
```
Can y'all confirm the issue? Is there any additional information I need to 
provide to get the status updated?
```

Original comment by `r...@hines57.com` on 29 Jun 2011 at 2:13

**Comment by GoogleCodeExporter** [comment]
```
Ping ... anything I can do to help with this?
```

Original comment by `Robert.S...@gmail.com` on 24 Oct 2011 at 8:24

**Comment by GoogleCodeExporter** [comment]
```
Fixed in upcoming version 1.4.7 scheduled for release November 2011
```

Original comment by `erik.eng...@devaldi.com` on 2 Nov 2011 at 2:44
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Whooot! Thanks!
```

Original comment by `r...@hines57.com` on 2 Nov 2011 at 2:54
